src/ClientResolver.php

```diff
@@ -287,6 +287,20 @@ class ClientResolver
             'default'   => false,
             'fn'        => [__CLASS__, '_apply_suppress_php_deprecation_warning']
         ],
+        'ignore_configured_endpoint_urls' => [
+            'type'      => 'value',
+            'valid'     => ['bool'],
+            'doc'       => 'Set to true to disable configured endpoint urls.',
+            'fn'        => [__CLASS__, '_apply_ignore_configured_endpoint_urls'],
+            'default'   => [__CLASS__, '_default_ignore_configured_endpoint_urls'],
+        ],
+        'endpoint_url' => [
+            'type'      => 'value',
+            'valid'     => ['string'],
+            'doc'       => 'The full URI of the webservice. This is only required when connecting to a custom endpoint (e.g., a local version of S3). This will take precedence over the `endpoint` configuration option.',
```

Comment:
This doc could be a bit clearer about how this differs from endpoint

Reply:
It doesn't really differ from `endpoint`.  It's essentially the same thing with different a different name and configuration locations.  I can add that though.